content/references/gamification/tables.md

```diff
@@ -0,0 +1,105 @@
+---
+kind: reference
+sidebar: ranks-honor
+prev: /references/gamification/privileges/
+---
+
+# Leveling rewards and requirements
+
+## Rank requirements
+
+| User Rank | Required Score |
+| :-------- | -------------: |
+| 8 kyu     |              0 |
+| 7 kyu     |             20 |
+| 6 kyu     |             76 |
+| 5 kyu     |            229 |
+| 4 kyu     |            643 |
+| 3 kyu     |          1,768 |
+| 2 kyu     |          4,829 |
+| 1 kyu     |         13,147 |
+| 1 dan     |         35,759 |
+| 2 dan     |         97,225 |
+
+## Rank rewards
+
+| Kata Rank | Score Awarded |
+| :-------- | ------------: |
+| 8 kyu     |             2 |
+| 7 kyu     |             3 |
+| 6 kyu     |             8 |
+| 5 kyu     |            21 |
+| 4 kyu     |            55 |
+| 3 kyu     |           149 |
+| 2 kyu     |           404 |
+| 1 kyu     |         1,097 |
```

Comment:
For these two, it might be better to show on the same page. These are small tables and not much to expand on.

Reply:
I am sorry, I do not follow: on the same as what, and on other than what, exactly?

Reply:
I thought these tables should be on the Ranks page instead of that page. These are small tables and require no further explanation. It breaks the flow by being on a separate page without gain.

I wish there was some easy way to embed partial Markdowns. Then we can have it on both places.